zerver/lib/webhooks/common.py

```diff
@@ -75,10 +77,27 @@ def check_send_webhook_message(
     user_profile: UserProfile,
     topic: str,
     body: str,
+    complete_event_type: Optional[str] = None,
     stream: Optional[str] = REQ(default=None),
     user_specified_topic: Optional[str] = REQ("topic", default=None),
+    only_events: Optional[List[str]] = REQ(default=None, json_validator=check_list(check_string)),
+    exclude_events: Optional[List[str]] = REQ(
+        default=None, json_validator=check_list(check_string)
+    ),
     unquote_url_parameters: bool = False,
 ) -> None:
+    if complete_event_type is not None:
+        # If complete_event_type is passed to this function, we will check the event type against user configured lists of only_events
+        # and exclude events. If the event does not satisfy the configuration, the function will return without doing anything.
+        # We match items in only_events and exclude_events using Unix shell-style wildcards.
```

Comment:
We should perhaps wrap comments at 80 characters, it makes it easier to read code on a dual window splitscreen setup.